Source/WebKitLegacy/WebCoreSupport/SocketStreamHandleImpl.h

```diff
@@ -51,17 +51,17 @@ class SocketStreamHandleImpl : public SocketStreamHandle {
 
     virtual ~SocketStreamHandleImpl();
 
-    WEBCORE_EXPORT static void setLegacyTLSEnabled(bool);
+    static void setLegacyTLSEnabled(bool);
 
-    WEBCORE_EXPORT void platformSend(const uint8_t* data, size_t length, Function<void(bool)>&&) final;
-    WEBCORE_EXPORT void platformSendHandshake(const uint8_t* data, size_t length, const std::optional<CookieRequestHeaderFieldProxy>&, Function<void(bool, bool)>&&) final;
-    WEBCORE_EXPORT void platformClose() final;
+    void platformSend(std::span<const uint8_t> data, Function<void(bool)>&&) final;
+    void platformSendHandshake(const uint8_t* data, size_t length, const std::optional<CookieRequestHeaderFieldProxy>&, Function<void(bool, bool)>&&) final;
```

Comment:
Why doesn't `platformSendHandshake` take a `span` now?

Reply:
It should. I'm just trying to keep patch size reasonable so I have to stop "somewhere" :)